src/common/cdomains/floatOps/floatOps.mli

```diff
@@ -7,7 +7,7 @@ type round_mode =
   | Down
 
 module type CFloatType = sig
-  type t
+  type t = float
```

Comment:
This should still be abstract, but `to_float` could just return `float` instead of `float option`.